registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -4999,6 +4999,46 @@
           "tokenManagerType": "mintBurn"
         }
       ]
+    },
+    "0x9afef659174ca5a4b0a182c8d6e85f4ea2f89c9bd0839a7181d989827ea93811": {
+      "tokenId": "0x9afef659174ca5a4b0a182c8d6e85f4ea2f89c9bd0839a7181d989827ea93811",
+      "deployer": "0x6229E0ddDBE2EC81Dd745C99659d52688a5BaE0f",
+      "originalMinter": null,
+      "prettySymbol": "GRAM",
+      "decimals": 18,
+      "originAxelarChainId": "Ethereum",
+      "tokenType": "canonical",
+      "deploySalt": "0x",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/gram.svg"
+      },
+      "deploymentMessageId": "",
+      "chains": [
+        {
+          "symbol": "GRAM",
+          "name": "GRAM Token",
+          "axelarChainId": "Ethereum",
+          "tokenAddress": "0x856D602E73545deAA1491a3726cF628d49f74F51",
+          "tokenManager": "0x2EBF71e6329Fe6702494ef72fFd1DA230b70F3A5",
```

Comment:
this manager does not seem to be [deployed](https://etherscan.io/address/0x2EBF71e6329Fe6702494ef72fFd1DA230b70F3A5)

Reply:
Yes. Ignore for now. Discussing with the team internally. Thanks